content/python/concepts/built-in-functions/terms/super/super.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'super()'
+Description: 'implicitly refer to the superclass'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Objects'
+  - 'Inheritance'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `super()` function is built into python and is used to refer to the the superclass of an object without specifically using the parent name.
+
+## Syntax
+
+```pseudo
+super().function()
+super(type) 
+```
+
+## `super()`
+
+`super()` returns a temporary object of the superclass, allowing the superclass’s methods to be called. This comes in handy when using [multiple inheritance](#multiple-inheritance).
+
+## Example
```

Comment:
```suggestion
## Codebyte Example

```

Let's allow one empty line above and below each header.

Reply:
Also let's rename this to "Codebyte Example" since Python is a supported language for [runnable code snippets](https://help.codecademy.com/hc/en-us/articles/1260806285369-Codebytes) (which is compatible with Codecademy Docs 😄)

Reply:
Updated